src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -44,7 +44,7 @@ public function find($includeArgs = true)
         }
 
         // PHP_BINARY return the current sapi executable
-        if (defined('PHP_BINARY') && PHP_BINARY && in_array(PHP_SAPI, array('cli', 'cli-server', 'phpdbg')) && is_file(PHP_BINARY)) {
+        if (PHP_BINARY && in_array(PHP_SAPI, array('cli', 'cli-server', 'phpdbg')) && is_file(PHP_BINARY)) {
```

Comment:
according to https://github.com/symfony/symfony/pull/16605#issuecomment-160131936 I'd say the constant might not be an empty string when using PHP FPM